app/controllers/index.rb

```diff
@@ -0,0 +1,35 @@
+require 'sinatra'
+
+# get '/' do
+#   "Stephanie"
+# end
+
+#create
+post '/sealions' do
+  Sealion.create(params)
```

Comment:
Good work! Let's say we wanted to redirect to the sealion we've just created show route instead! How would the redirect uri change?